cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -30,6 +30,8 @@ spec:
       - image: gcr.io/stackdriver-agents/stackdriver-metadata-agent:{{ metadata_agent_version }}
         imagePullPolicy: IfNotPresent
         name: metadata-agent
+        command: ["/opt/stackdriver/metadata/sbin/metadatad"]
```

Comment:
Please use list syntax consistent with the rest of the file (each element in separate line preceded with a dash).

Reply:
Done. Merged the args into the command list.